### PR TITLE
Update .env.example

### PR DESCRIPTION
**Pull request by otmane-tech**

### CODE REVIEW

File: frontend/.env.example [hunk 1]
```diff
@@ -2,4 +2,3 @@ NEXT_PUBLIC_STRAPI_API_TOKEN=your-api-token
 NEXT_PUBLIC_PAGE_LIMIT=6
 NEXT_PUBLIC_STRAPI_FORM_SUBMISSION_TOKEN=your-form-submission-token
 NEXT_PUBLIC_STRAPI_API_URL=http://localhost:1337
-
```
Comment (elhaddajiOtmane) [inline frontend/.env.example:5]: thanks

File: frontend/.env.example [hunk 2]
```diff
@@ -2,4 +2,3 @@ NEXT_PUBLIC_STRAPI_API_TOKEN=your-api-token
 NEXT_PUBLIC_PAGE_LIMIT=6
```
Comment (elhaddajiOtmane) [inline frontend/.env.example:2]: make it 10

**Review by elhaddajiOtmane — APPROVED** [review]
good

File: frontend/.env.example (hunk 2 above)
Comment (otmane-tech) [reply frontend/.env.example:2]: ds

File: frontend/.env.example [hunk 3]
```diff
@@ -2,4 +2,3 @@ NEXT_PUBLIC_STRAPI_API_TOKEN=your-api-token
...
 NEXT_PUBLIC_STRAPI_FORM_SUBMISSION_TOKEN=your-form-submission-token
```
Comment (elhaddajiOtmane) [inline frontend/.env.example:3]: good

**Review by elhaddajiOtmane — COMMENTED** [review]
good

**Pull request merged**